src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
We know what we're talking about here. Don't take half measures. Overwrite or verify checksums. No shortcuts.